src/components/Calculator.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+function Calculator() {
```

Comment:
Great job @Rana-Mudassir 🤝

- [ ] As stated in the project requirements, kindly remove all the boilerplate that got created by the CRA (texts, images, and styles). You may delete these files `logo.svg`, `App.test.js`, `reportWebVitals.js`, `setupTests.js`.
- [ ] Kindly ensure `Calculator.js` should have at least 1 child component.